File: BancoJava/src/Usuario.java
public class Usuario {
	public String Name;
	public String Psword;
	public int Age;
	public String Address;	
	public double Saldo = 1000.0f;
	
	public Usuario(String nome, String senha, int idade, String endereco) {
		Name = nome;
		Psword = senha;
		Age = idade;
		Address = endereco;
	}

}

File: BancoCS/BancoCS/Usuario.cs
using System;

namespace BancoCS
{
    class Usuario
    {
        public string nome;
        public string senha;
        public int idade;
        public string endereco;

        public Usuario(string _nome, string _senha, int _idade, string _endereco)
        {
            nome = _nome;
            senha = _senha;
            idade = _idade;
            endereco = _endereco;
        }
    }
}
